### PR TITLE
[docs] Update network policy docs

### PR DESCRIPTION
**Pull request by npinaeva**
Add EgressQos and Metrics doc links to the readme

Signed-off-by: Nadia Pinaeva <npinaeva@redhat.com>

### CODE REVIEW

File: docs/network-policy.md [hunk 1]
```diff
@@ -60,61 +60,61 @@ spec:
...
+external_ids        : {name=a16982411286042166782_ingressDefaultDeny}
+name                : a16982411286042166782_ingressDefaultDeny
+ports               : []
 ```
 
-Two ACLs (four total) are added to each PortGroup:
+Two ACLs (four total) are added to each DefaultDeny PortGroup:
 
-1. a drop policy with `priority=1000` and `direction=to-lport`
+1. a drop policy with `priority=9100`, `direction=to-lport`, and `name=<namespace>_<e/in>gressDefaultDeny`
```
Comment (trozet) [inline docs/network-policy.md:84]: 9100? Default deny priority is 1000.

Comment (trozet) [reply docs/network-policy.md:84]: maybe this is after your refactor of ACLs or something? In which case we should hold this PR until that goes in?

Comment (npinaeva) [reply docs/network-policy.md:84]: Yes exactly - I forgot to add the dependency. Let's wait for now

**Pull request closed without merging**